modules/apps/change-tracking/change-tracking-web/src/main/java/com/liferay/change/tracking/web/internal/display/context/ViewRelatedEntriesDisplayContext.java

```diff
@@ -123,14 +123,15 @@ public <T extends BaseModel<T>> Map<String, Object> getReactData()
 				_ctCollectionId, ctSQLMode, ctEntry.getModelClassNameId(),
 				ctEntry.getModelClassPK());
 
-			if (!isShowHideable() &&
+			if (!isShowHideable() && (ctModel != null) &&
```

Comment:
We should try to get the model from production since no ctModel exists in the publication for a deletion ctEntry. This way we can properly check whether the change is hideable.

```
if (ctModel == nul) {
	ctModel = _ctDisplayRendererRegistry.fetchCTModel(
		ctEntry.getModelClassNameId(), ctEntry.getModelClassPK()
	);
}
```